src/ncp/ncp_buffer.hpp

```diff
@@ -381,36 +382,36 @@ class NcpFrameBuffer
     otError         OutFramePrepareMessage(void);
     otError         OutFrameFillMessageBuffer(void);
 
-    uint8_t * const  mBuffer;                    // Pointer to the buffer used to store the data.
-    uint8_t * const  mBufferEnd;                 // Points to after the end of buffer.
-    const uint16_t   mBufferLength;              // Length of the the buffer.
+    uint8_t * const mBuffer;                    // Pointer to the buffer used to store the data.
```

Comment:
This was automatically changed with the "make pretty" prior to committing...  I can undo if needed...